cmake/TargetCapnpSources.cmake

```
@@ -60,15 +60,23 @@ function(target_capnp_sources target include_prefix)
     "IMPORT_PATHS"  # multi_value_keywords
   )
 
-  if(NOT TARGET Libmultiprocess::mpgen)
-    message(FATAL_ERROR "Target 'Libmultiprocess::mpgen' does not exist.")
+  set(MPGEN_BINARY "")
+  if(EXTERNAL_MPGEN)
+    set(MPGEN_BINARY "${EXTERNAL_MPGEN}")
+    if(NOT EXISTS "${MPGEN_BINARY}")
+      message(FATAL_ERROR "EXTERNAL_MPGEN: \"${MPGEN_BINARY}\" does not exist.")
+    endif()
+  elseif(TARGET Libmultiprocess::multiprocess)
+    set(MPGEN_BINARY $<TARGET_FILE:Libmultiprocess::mpgen>)
```

I'm fairly certain that a generation expression isn't necessary here:
```suggestion
    set(MPGEN_BINARY Libmultiprocess::mpgen)
```